api/managementpb/checks.proto

```diff
@@ -53,4 +78,18 @@ service SecurityChecks {
       body: "*"
     };
   }
+  // ListSecurityChecks returns list of available Security Checks and their status.
+  rpc ListSecurityChecks(ListSecurityChecksRequest) returns (ListSecurityChecksResponse) {
+    option (google.api.http) = {
+      post: "/v1/management/SecurityChecks/List"
+      body: "*"
+    };
+  }
+  // ChangeSecurityChecks enables/disables Security Checks by name.
+  rpc ChangeSecurityChecks(ChangeSecurityChecksRequest) returns (ChangeSecurityChecksResponse) {
```

Comment:
this name is a bit ambiguous, it reads as if we are changing something in the checks / check script itself rather than just one field of a check, I preferred the names you used previously `Update...`/`Toggle..` either of them would be better than this

Reply:
ok I missed this comment https://github.com/percona/pmm/pull/588#discussion_r505663440 didn't know it was a requested change, never mind